### ISSUE TITLE
access: need a better mechanism for making Group files readable to others.

### ISSUE THREAD

**Issue by edpin** [upspin/upspin#73]
Currently, if user A wants to include a group file from user B, it's not sufficient that A can read B's Group directory. It is also necessary that A's DirServer be able to see it so it can parse it.

The current work-around for group files to work is to put an access file under /Group giving read access to the DirServer. But that's ugly.

In discussing with Dave, we came up with a proposal:
- Group files whose names start with a lowercase letter ("family") are not visible by anyone other than the owner (or to people explicitly listed in a ruling Access file).
- Group files whose names start with an uppercase letter ("Family") are world-readable.

Both these rules apply on top of Access file rules, meaning one cannot List group files unless they were granted List access explicitly. And so on for other rights

In summary: the exported group filenames is a simple add-on rule to _read_ Access permissions. 

Thoughts?
